osu.Game/Screens/Edit/Compose/Components/SelectionHandler.cs

```diff
@@ -254,23 +250,18 @@ private void deleteSelected()
         /// <summary>
         /// Updates whether this <see cref="SelectionHandler"/> is visible.
         /// </summary>
-        internal void UpdateVisibility()
+        private void updateVisibility()
```

Comment:
could potentially call this with `Scheduler.AddOnce` to reduce multiple calls? would need to check it behaves correctly still.

Reply:
Hmm, good call, sure. Will try.